components/uniset/src/uniset.rs

```diff
@@ -143,6 +143,13 @@ impl UnicodeSet {
         self.contains_query(query as u32).is_some()
     }
 
+    /// Returns the inversion list as a slice
+    ///
+    /// Public only to the crate, not exposed to public
```

Comment:
No need to indicate that, it's visible in source and in docs.

Reply:
agreed and changed